common/buildings/wc_vine_quilboar_buildings.txt

```diff
@@ -0,0 +1,231 @@
+﻿#Quilboar vined thorns
+quilboar_thorns_01 = {
+	type_icon = "icon_structure_corpse_of_agamaggan.dds"
+
+	construction_time = quick_construction_time
+
+	can_construct_potential = { always = no }
+
+	levy = 100
+	county_modifier = {
+		supply_limit = 125
+	}
+
+	character_modifier = {
+		same_faith_opinion = 2
+		monthly_piety_gain_mult = 0.05
+	}
+	province_modifier = {
+		stationed_caster_siege_value_add = 1
+		stationed_skirmishers_siege_value_add = 1
```

Comment:
```suggestion
		stationed_caster_siege_value_add = 0.05
		stationed_skirmishers_siege_value_add = 0.05
```
wow wow wait. Thats too much. Martial lifestyle perk gives for example `0.1` only. Take example from `workshops_01`
lvl1=0.05
lvl2=0.1
lvl3=0.15
...
lvl8=0.4